samples/bluetooth/hci_uart/boards/bbc_microbit.conf

```diff
@@ -2,5 +2,7 @@ CONFIG_MAIN_STACK_SIZE=512
 CONFIG_IDLE_STACK_SIZE=256
 CONFIG_ISR_STACK_SIZE=512
 CONFIG_BT_MAX_CONN=10
-CONFIG_BT_RX_BUF_LEN=76
-CONFIG_BT_DISCARDABLE_BUF_SIZE=45
+# Revert prj.conf values to default
```

Comment:
I don't understand what you meant here. Are you saying that the comment is not clear enough?

The point here is to revert the changes made by the hci_uart/prj.conf to set them back to their Kconfig default values.
Because these boards don't fit in RAM otherwise.

Reply:
Thanks for the explanation